### DISCUSSION TITLE
Permission to use repository or addition of license

### DISCUSSION THREAD

**Discussion by Origor** [jmforsythe/Git-Heat-Map#12] [General]
Hi @jmforsythe, 

I wanted to prepare some material for some new summer workers/interns at a company I work for and was really interested in use of your repository since it could fit the project idea. I saw your comment in https://github.com/jmforsythe/Git-Heat-Map/pull/6#issuecomment-1424980568 when being asked about a License therefore this question. 

The idea is to get them to use your tool on a repository and from the visualization let them investigate some parts of the repository and from there try and play around with some ideas about re-designing, simplifying or some tracking. I've read that per default all rights are reserved, some information in [GitHub's Docs about Licensing](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository) about some different alternatives was helpful to understand this a bit clearer.

Well my question to you was if you had any plans on how you wanted to *license* your work and if not, I wanted to know if I could use it for the use case described above?

Cheers!

**Comment by jmforsythe** [comment]
I believe you're correct that all rights are reserved by default. I have not currently decided on a license.
Absolutely you can use this project for the use case described.

**Reply by Origor** [reply]
Thank you for the quick response, have a nice one.
